README.md

```diff
@@ -100,137 +66,80 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Use of PostgreSQL for databases and tables.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Coming soon](https://github.com/Nkaleth/Vet_clinic_database)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+-visual studio
+-github
+-PosgreSQL 15
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  mkdir Vet_clinic_database
+  cd Vet_clinic_database
+  git clone git@github.com:Nkaleth/Vet_clinic_database.git
 ```
---->
+-
```

Comment:
- [ ] As I mentioned in the previous comment think it would be better if you add a command that accesses the cloned directory after cloning your project. (i.e. `cd Vet_clinic_database`) to make your documentation more descriptive and professional. You can leave your old changes then the ``cd Vet_clinic_database`` command should come after the cloning command. **For example:**
```sh
  ...
  mkdir my-folder

  cd my-folder

  git clone git@github.com:Nkaleth/Vet_clinic_database.git

> To access the clone directory, run:
 `cd Vet_clinic_database`
 ...
```